deploy/charts/trust-manager/README.md

```diff
@@ -56,6 +56,7 @@ Kubernetes: `>= 1.25.0-0`
 | image.repository | string | `"quay.io/jetstack/trust-manager"` | Target image repository. |
 | image.tag | string | `nil` | Target image version tag. Defaults to the chart's appVersion. |
 | imagePullSecrets | list | `[]` | For Private docker registries, authentication is needed. Registry secrets are applied to the service account |
+| namespace | string | `nil` | The namespace to deploy trust-manager resources into. |
```

Comment:
I think you need to a an empty default (value), like it's done is cert-manager:

https://github.com/cert-manager/cert-manager/blob/9fd770a011815d8499a7fb35a6387982ec909762/deploy/charts/cert-manager/values.yaml#L98-L101

Also, this file is a generated file - generated with helm-docs. Which needs to be regenerated with `make update-helm-docs` after you have changed the values file.

Reply:
Good catch, I believe I have covered those requested changes in 98cdbd3.